Makefile

```diff
@@ -24,23 +34,23 @@ lint:
 .PHONY: test
 test:
 	@echo "hashmap freelist test"
-	TEST_FREELIST_TYPE=hashmap go test -v ${TESTFLAGS} -timeout 30m
-	TEST_FREELIST_TYPE=hashmap go test -v ${TESTFLAGS} ./internal/...
-	TEST_FREELIST_TYPE=hashmap go test -v ${TESTFLAGS} ./cmd/bbolt
+	TEST_FREELIST_TYPE=hashmap TEST_ENABLE_STRICT_MODE=${TESTFLAGS_ENABLE_STRICT_MODE} go test -v ${TESTFLAGS} -timeout ${TESTFLAGS_TIMEOUT}
+	TEST_FREELIST_TYPE=hashmap TEST_ENABLE_STRICT_MODE=${TESTFLAGS_ENABLE_STRICT_MODE} go test -v ${TESTFLAGS} ./internal/...
+	TEST_FREELIST_TYPE=hashmap TEST_ENABLE_STRICT_MODE=${TESTFLAGS_ENABLE_STRICT_MODE} go test -v ${TESTFLAGS} ./cmd/bbolt
```

Comment:
Adding more env vars makes the Makefile harder to read. Can you consider utilizing `.EXPORT_ALL_VARIABLES` target if that works for this case?

https://stackoverflow.com/a/49524393/242451

Reply:
thx & updated.